setup:
    prefix: 'j2s7s300_driver'
    model_filename: "jaco_dynamics"
    object_centers: {'HUMAN_CENTER': [-0.6,-0.55,0.0], 'LAPTOP_CENTER': [-0.7929,-0.1,0.0]}
    feat_list: ["efficiency", "table", "laptop"]
    demo_spec: "all_0" # "simulate", "all" followed by task number, ID number followed by task number
    demo_dir: "/data/demonstrations/demos/"

sim:
    task:
        feat_weights: [1.0,0.0,0.0]
        start: [104.2, 151.6, 183.8, 101.8, 224.2, 216.9, 310.8]
        goal: [210.8, 101.6, 192.0, 114.7, 222.2, 246.1, 322.0]
        goal_pose: None # [-0.46513, 0.29041, 0.69497]
        T: 20.0
        timestep: 0.5
    planner:
        # These settings have been tuned for trajopt planner.
        type: "trajopt"
        max_iter: 50
        num_waypts: 5

learner:
    trajs_path: "/data/traj_sets/traj_rand_merged_H.p"
    betas_list: [0.01, 0.03, 0.1, 0.3, 1.0, 3.0, 10.0, 30.0, 100.0]
    weight_vals: [0.0, 0.5, 1.0] # Per feature theta options.
    FEAT_RANGE: {'table':1.0, 'coffee':1.0, 'laptop':1.6, 'human':1.6, 'efficiency':0.01}


